chapters/infrastructure/processing/reading/README.md

```diff
@@ -1,12 +1,41 @@
+---
+title: Content Processing
+---
+
 ### Content Processing
 
-Owner: Vlad Năstase
-Reviewer: Teo Duțu
+Content organized in a Version Control System like `git`, and published to a `GitHub` repository is suitable for easier contributions.
+In order to deliver it to the end user/learner, this content has to be processed and rendered in an easier to navigate form.
+
+The types of content that need to be processed vary with course requirements and require different actions in order to be rendered.
+These types include, but are not limited to:
+
+- slides
+- reading material
+- quizes
+- various types of media content.
+
+The rendered output that we recommend is a website generated using a static website generator.
+The usage of [Docusaurus](https://docusaurus.io/) is also recommended, but other static website generators (Hugo, Docsy, Sphinx) can be used.
+The rendered website can embed materials such as slides and quizes, and can also refer the end user/learner to the `GitHub` repository from which the content is generated (or to an auxiliary repository) for practical content, such as drills.
+
+Most static website generators have a documented structure that needs to be followed for the website to render.
+This can be incompatible with the way content is organized for easier collaboration.
+
+To facilitate content processing with the scope of being published, we have developed the [`Open Education Builder`](https://open-education-hub.github.io/openedu-builder/).
+
+#### Open Education Builder
+
+The Open Education Builder is an application that helps with the organization, transformation and output of content.
+It is built with a modular architecture in mind, the smallest building block being the `plugin`.
+A `plugin` represents a simple script that transforms the original content in some way.
+A few examples of such plugins are the ones developed for the technologies recommended in our methodology:
 
-process content to render it to the end user (learners)
+- `revealmd` - builds multiple slides that are based on `reveal-md`;
+- `docusaurus` - helps with organization of content in a manner that is compatible with Docusaurus.
 
-different types of content to be homogenized
+The plugins are configured through a central configuration file (`config.yaml`), present at the root of your content repository.
 
-site + file archives / repositories to be present to learners
+The main way to use the Open Education Builder is by customizing the default actions and configuration present in the [`oer-template`](https://github.com/open-education-hub/oer-template) repository.
 
-OpenEdu builder
+For configuration options, advanced usage and contributions, please refer to the [documentation](https://open-education-hub.github.io/openedu-builder/) and the [Open Education Builder repository](https://github.com/open-education-hub/openedu-builder)
```

Comment:
You should give some examples here. Otherwise, the config file is going to be difficult to use. Ideally, you would have:
- examples of how to use the builder (both manually and as an action)
- examples of how to use `config.yml` and its interaction with the builder.